packages/react-instantsearch/src/connectors/connectToggle.js

```diff
@@ -56,7 +56,7 @@ function cleanUp(props, searchState, context) {
  * @propType {boolean} [defaultRefinement=false] - Default searchState of the widget. Should the toggle be checked by default?
  * @providedPropType {function} refine - a function to toggle a refinement
  * @providedPropType {function} createURL - a function to generate a URL for the corresponding search state
- * @providedPropType {boolean} currentRefinement - the refinement currently applied
+ * @providedPropType {boolean} currentRefinement - `true` when the refinement is applied, `false` otherwise
```

Comment:
I think the currentRefinement is `[]` or [{...stuff, currentRefinement: true}]` makes more sense

Reply:
I don't understand? The `currentRefinement` provided by the connector is a boolean value. The shape that you describe is the one provided by the `getMetadata` function I think.

https://github.com/algolia/react-instantsearch/blob/9a8a154ca937bf7fa9df9b41b3a79b86a41613ed/packages/react-instantsearch/src/components/Toggle.js#L19-L31

Reply:
oh yes that's true, never mind then 😂